### DISCUSSION TITLE
Bug: tool json-to-yaml: 语法错误标示异常

### DISCUSSION THREAD

**Discussion by alctny** [he3-app/he3-feedback#143] [Bug Report]
### Version
json-to-yaml 1.0.1

### Environment
Linux v1.3.2

### Steps to reproduce

在左侧的 JSON 输入栏中输入如下内容(第 4 行存在语法错误)

```json
{
  "a": [{
    "a":1
  },,
  {
    "a":2
  }]
}
```

### What is expected?

这时候虽然会提示语法错误，但是不会在有语法错误所在行标红。

这是正常提示时候的样子
![正常提示](https://user-images.githubusercontent.com/44720162/233241073-1a8bfe4d-1ff8-43f6-9de5-c88e62eddd82.png)

但是下面两种情况虽然有提示语法错误，却没有在具体行进行标注

![未正常提示](https://user-images.githubusercontent.com/44720162/233241035-6a1ef60a-c7af-4725-a92a-747290731c6f.png)

![未正常提示-2](https://user-images.githubusercontent.com/44720162/233241326-83a0d7d8-4570-4b38-a266-b91bbe520bc1.png)

**Comment by alctny** [comment]
这问题在 Windows 上同样存在

**Comment by lyzhang1999** [comment]
感谢您的反馈，已经建立相应的 issues 跟进这个问题：https://github.com/he3-app/he3-feedback/issues/144 ，请关注。
